include/glow/IR/IRGen.h

```diff
@@ -0,0 +1,58 @@
+#ifndef GLOW_IR_IRGEN_H
+#define GLOW_IR_IRGEN_H
+
+#include "glow/Backends/Backend.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Graph/Nodes.h"
+#include "glow/IR/IR.h"
+#include "glow/IR/IRBuilder.h"
+
+#include <unordered_set>
+
+//===----------------------------------------------------------------------===//
+//              IRGen visitor - the code that generates the IR.
+//===----------------------------------------------------------------------===//
+
+namespace glow {
+
+/// A helper class for visiting and generating the dotty file from the graph.
```

Comment:
This comment is wrong -- I see it was copied from IRGen.cpp, but can you fix it while you're moving the class too?